src/cpu/flexcpu/simple_dataflow_thread.hh

```diff
@@ -208,8 +209,11 @@ class SDCPUThread : public ThreadContext
      * related events, such as attemptFetch to kick off any upcoming tasks
      * necessary to complete that instruction. May issue an instruction
      * immediately if no new information is needed to set up a new instruction.
+     *
+     * @param nextPC The PC the system should be at for fetching and executing
+     *  the upcoming instruction.
      */
-    void advanceInst();
+    void advanceInst(TheISA::PCState next_pc);
```

Comment:
I'm glad you changed this to take a PC. It makes more sense this way :).

Reply:
Yea it was time to do it when the next pc didn't always come from the same place